edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -34,11 +34,6 @@
       {% endblocktrans %}
     </p>
     <p>
-      {% blocktrans %}
-        You will be asked to verify your identity before you begin the exam. Make sure you have valid photo identification, such as a driver's license or passport, before you continue.
```

Comment:
@catong How's this?

>         You will be asked to verify your identity as part of the proctoring exam set up.
        Make sure you are on a computer with a webcam, and that you have valid photo identification
        such as a driver's license or passport, before you continue.